lib/stringToChars.js

```diff
@@ -0,0 +1,11 @@
+
+function stringToChars(string) {
+  let arrayChars = [];
+  for (let i of string) {
+    arrayChars.push(i);
+  }
+  return (arrayChars);
+}
+stringToChars('Hello');
```

Comment:
This line will be run every time you `require` this file. You'll want to remove it.